### PR TITLE
Update Fork and Branches challenges

### PR DESCRIPTION
**Pull request by jlord**
These two challenges are some of the ones people have the most trouble with. This PR updates the text to hopefully be more clear and it updates the verify step in the Forks challenge to explicitly check that the user has set up their `origin` remote to point to their fork.

Closes https://github.com/jlord/git-it-electron/issues/143
Closes https://github.com/jlord/git-it-electron/issues/133
Closes https://github.com/jlord/patchwork/issues/11600

### CODE REVIEW

**Comment by Papamani** [comment]
how

**Comment by jencaza33** [comment]
Gonna need you to dumb it down a little more please.  What exactly needs to be input for branch name when you git push origin?.. when I viewed all my remotes are the same URL.. where do I find the URL for the branch I created?
